### PR TITLE
Cells Sliders

### PR DESCRIPTION
**Pull request by mccalluc**
@john-conroy : Code review
@tsliaw : Design review: Are sliders that look like this ok? (Would like to keep to the defaults where possible.) Width/positioning will change when the whole page is handled.
Screen Shot 2021-06-23 at 3 58 29 PM

### CODE REVIEW

File: context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx [hunk 1]
```diff
@@ -38,29 +39,36 @@ function DatasetsSelectedByGene(props) {
...
     setFields[name](event.target.value);
   }
 
   return (
     <Paper>
       <TextField label="gene name" value={geneName} name="geneName" variant="outlined" onChange={handleChange} />
-      <TextField
-        label="min gene expression"
+
+      <p id="min-gene-expression-label">Minimun gene expression</p>
```
Comment (tsliaw) [inline context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx:50]:
```suggestion
      <p id="min-gene-expression-label">Minimum gene expression</p>
```

File: context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx [hunk 2]
```diff
@@ -38,29 +39,36 @@ function DatasetsSelectedByGene(props) {
...
+        max={100}
+        onChange={(e, value) => {
+          setMinGeneExpression(value);
+        }}
+        aria-labelledby="min-gene-expression-label"
       />
-      <TextField
-        label="min cell percentage"
+
+      <p id="min-cell-percentage-label">Minimun cell percentage</p>
```
Comment (tsliaw) [inline context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx:61]:
```suggestion
      <p id="min-cell-percentage-label">Minimum cell percentage</p>
```

**Review by tsliaw — COMMENTED** [review]
Are you planning to add number labels to these sliders in a different PR? The style of the slider looks fine since it's from Materials, but the sliders do need min/max number labels and when dragging/hovering, the current number should appear. 

Screen Shot 2021-06-23 at 1 28 11 PM

[Materials](https://material.io/components/sliders#discrete-sliders)

File: context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx (hunk 1 above)
Comment (john-conroy) [inline context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx:50]: Use label instead of p?

Comment (john-conroy) [reply context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx:50]: I think MUI has a label component for inputs/forms.

File: context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx [hunk 3]
```diff
@@ -38,29 +39,36 @@ function DatasetsSelectedByGene(props) {
...
     const setFields = {
       geneName: setGeneName,
-      minGeneExpression: setMinGeneExpression,
-      minCellPercentage: setMinCellPercentage,
     };
     setFields[name](event.target.value);
   }
 
   return (
     <Paper>
```
Comment (john-conroy) [inline context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx:47]: Should we use a form here? There are differing opinions on their necessity.

Comment (mccalluc) [reply context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx:47]: I'm hoping that's something you can sort out when you take these little pieces and make the real thing out of them!

File: context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx (hunk 1 above)
Comment (mccalluc) [reply context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx]: Sure, I'll use that instead ... but John is going to have a lot of these to fix, in any event.

**Comment by mccalluc** [comment]
Treating "comment" as "good-to-merge-with-tweaks", so I'll merge.

**Pull request merged**